scripts/security/cve-template.json

```diff
@@ -0,0 +1,111 @@
+{
+    "CVE_data_meta": {
+        "ASSIGNER": "vulnerabilities@zephyrproject.org",
+        "DATE_PUBLIC": "2020-05-01T00:00:00.000Z",
+        "ID": "CVE-2020-XXXXX",
+        "STATE": "PUBLIC",
+        "TITLE": ""
+    },
+    "affects": {
+        "vendor": {
+            "vendor_data": [
+                {
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "zephyr",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_affected": ">=",
+                                            "version_value": "1.14.0"
+                                        }
+                                    ]
+                                }
+                            }
+                        ]
+                    },
+                    "vendor_name": "zephyrproject-rtos"
+                }
+            ]
+        }
+    },
+    "credit": [
```

Comment:
Is this a valid attribute ? I didn't see it.

Reply:
This json is a direct copy of one generated by the vulnogram tool.

Reply:
will look it, looking the scheme I didn't see this property